motif_inference/unite_motifs_of_biological_condition.py

```diff
@@ -87,12 +87,15 @@ def unite_clusters(motif_inference_output_path, meme_file, biological_condition,
             clusters_to_combine.append(cluster_names)
 
     logger.info(f'Sorting clusters by rank...')
+    half_num_samples = round(len(sample_names)/2)
     # sort the sublist such that the first one will contain the highest copy number, etc...
-    clusters_to_combine.sort(key=lambda clusters: sum(get_cluster_size_from_name(cluster) for cluster in clusters), reverse=True)
+    clusters_to_combine.sort(key=lambda clusters: (len(set([get_number_samples_from_name(cluster, biological_condition) for cluster in clusters])), sum(get_unique_members_from_name(cluster) for cluster in clusters), sum(get_cluster_size_from_name(cluster) for cluster in clusters)), reverse=True)
```

Comment:
It is better to have this option as a flag (can be default) in case we want to return to previous sorting.
This is important to be able to reproduce previous results